### PR TITLE
Improvements for RC releases

### PR DESCRIPTION
**Pull request by FilipB**
* fixing minor version for strings with more '.'
* adding useful flags for gh release create cmd

### CODE REVIEW

File: Makefile.core.mk [hunk 1]
```diff
@@ -20,7 +20,7 @@ OLD_VARS := $(.VARIABLES)
 -include Makefile.vendor.mk
 
 VERSION ?= 0.2.0
-MINOR_VERSION := $(shell v='$(VERSION)'; echo "$${v%.*}")
+MINOR_VERSION := $(shell echo "$${VERSION}" | cut -f1,2 -d'.')
```
Comment (dgn) [inline Makefile.core.mk:23]:
```suggestion
MINOR_VERSION := $(shell echo "${VERSION}" | cut -f1,2 -d'.')
```

This will make it work

Comment (FilipB) [reply Makefile.core.mk]: Thank you

**Review by dgn — APPROVED** [review]

**Pull request merged**